src/test/java/com/google/api/codegen/testdata/discogapic/simplecompute.v1.json

```diff
@@ -948,6 +948,28 @@
   },
   "dummyObjects": {
    "methods": {
+    "getResource": {
+     "id": "compute.dummyObjects.getResource",
+     "path": "{project}/dummyObjects/{resource}",
```

Comment:
This file change introduces a test case for a resource path that would be rendered as a class called `ResourceName`.